docs/src/main/tut/decimal-migration-guide.md

```diff
@@ -6,11 +6,11 @@ permalink: /docs/rpc/decimal-migration-guide
 
 # Migration guide for decimal types (available from 0.15.1)
 
-## Intended audience
+## Intended audience
 
 This guide is only for projects that are in a prior version to `0.15.1` **and** have decimals in some of their protocols **and** want to serialize those decimals following the avro specs.
 
-If you're starting a new project, you can use safely tagged `BigDecimal`s. Check the [Custom codecs section in Core concepts](/docs/rpc/core-concepts#custom-codecs) and [Plugin Settings section in IDL Generation](/docs/rpc/idl-generation#plugin-settings) for more information.
+If you're starting a new project, you can use safely tagged `BigDecimal`s. Check the [Custom codecs section in Core concepts](/mu/docs/rpc/core-concepts#custom-codecs) and [Plugin Settings section in IDL Generation](/mu/docs/rpc/idl-generation#plugin-settings) for more information.
```

Comment:
Maybe using relative paths would fix the issue? -> `docs/rpc/core-concepts#custom-codecs`

Reply:
I can't say but now I see that all internal links are broken in the docs. Do you know how to check if the relative paths work?

Reply:
Running the microsite locally, the result should be the same as in gh-pages.